### ISSUE TITLE
[Bug] Cannot install on windows 10

### ISSUE THREAD

**Issue by djdecoit** [deezer/spleeter#431]
## Description
I installed anaconda following the wiki here, without using admin privileges as mentioned and followed along for everything, it was installed properly as i checked that too. Then after opening command prompt and powershell both,I tried installing spleeter through the mentioned process, but I get this particular error where it reads "failed with initial frozen solve, trying with flexible solve" and the subsequent step fails too, and this cycle repeats itself a couple of times, for around 30 minutes when I finally close the command prompt.

Now, I once opened Anaconda navigator and then i followed the installation process by launching command prompt from navigator, there story was slightly different and a whole lot of checking was going on which goes above my head, so I would be pasting everything in the section below from that instance.

## Step to reproduce

1. Installed using `Anaconda`
2. Run as `User`
3. Got `failed with initial frozen solve` error

## Output

```bash
(base) C:\Users\Ankur> conda install -c conda-forge spleeter
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: \
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
Examining xlrd:   7%|████▋                                                            | 22/302 [01:07<09:14,  1.98s/it]/Examining mccabe:  38%|███████████████████████▍                                      | 114/302 [02:35<02:03,  1.52it/s]|Examining lxml:  47%|██████████████████████████████▎                                 | 143/302 [03:05<02:20,  1.13it/s]\Examining python-jsonrpc-server:  48%|██████████████████████▍                        | 144/302 [03:36<02:17,  1.15it/s]\Examining snowballstemmer:  53%|███████████████████████████▉                         | 159/302 [03:54<02:34,  1.08s/it]/Examining conda-verify:  55%|██████████████████████████████▌                         | 165/302 [03:55<01:06,  2.05it/s]|Examining html5lib:  56%|█████████████████████████████████▊                          | 170/302 [03:59<02:05,  1.05it/s]\Examining partd:  57%|███████████████████████████████████▉                           | 172/302 [03:59<01:19,  1.63it/s]|Examining pep8:  58%|█████████████████████████████████████                           | 175/302 [04:00<00:50,  2.52it/s]/Examining joblib:  65%|████████████████████████████████████████▍                     | 197/302 [04:07<00:40,  2.60it/s]|Examining pkginfo:  69%|██████████████████████████████████████████▏                  | 209/302 [04:43<03:52,  2.50s/it]\Examining unicodecsv:  76%|███████████████████████████████████████████▉              | 229/302 [04:56<00:51,  1.41it/s]|Examining python-dateutil:  77%|████████████████████████████████████████▋            | 232/302 [04:57<01:33,  1.34s/it]\Examining pip:  79%|███████████████████████████████████████████████████▏             | 238/302 [05:00<00:45,  1.42it/s]\Examining psutil:  80%|█████████████████████████████████████████████████▋            | 242/302 [05:05<01:08,  1.14s/it]-Examining jupyterlab_server:  91%|██████████████████████████████████████████████▌    | 276/302 [05:36<00:19,  1.33it/s]\Examining yaml:  98%|██████████████████████████████████████████████████████████████▋ | 296/302 [06:08<00:18,  3.02s/it]-Examining conflict for python-jsonrpc-server winpty tbb astropy pyreadline nose sphinxcontrib-applehelp xlwt pyparsing /Examining conflict for pyflakes jinja2 pywinpty anaconda nose conda conda-build jupyterlab_server ipython flake8 joblib/Examining conflict for python-jsonrpc-server tbb astropy pyreadline nose xlwt pyparsing conda-build Examining conflict for glob2 anaconda conda-build:   5%|█▍                          | 16/302 [04:38<1:25:29, 17.93s/it]Examining conflict for glob2 anaconda conda-build:   6%|█▌                          | 17/302 [04:38<1:26:11, 18.14s/it]Examining conflict for anaconda pytest-astropy distributed spyder psutil pytest-openfiles anaconda-navigator dask condaExamining conflict for Examining conflict for anaconda conda-package-Examining conflict for libtiff pillow failed                                                   /       |                            |                      | - 69s/i\ / thon flake8 paramiko ipyw|Examining conflict for beautifulsoup4 anaconda conda-build:  14%|██▉
     | 42/302 [07:21<42:43,  9.86s/i\
# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\cli\install.py", line 265, in install
        should_retry_solve=(_should_retry_unfrozen or repodata_fn != repodata_fns[-1]),
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\core\solve.py", line 117, in solve_for_transaction
        should_retry_solve)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\core\solve.py", line 158, in solve_for_diff
        force_remove, should_retry_solve)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\core\solve.py", line 281, in solve_final_state
        ssc = self._run_sat(ssc)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\common\io.py", line 88, in decorated
        return f(*args, **kwds)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\core\solve.py", line 808, in _run_sat
        should_retry_solve=ssc.should_retry_solve
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\common\io.py", line 88, in decorated
        return f(*args, **kwds)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\resolve.py", line 1316, in solve
        raise UnsatisfiableError({})
    conda.exceptions.UnsatisfiableError:
    Did not find conflicting dependencies. If you would like to know which
    packages conflict ensure that you have enabled unsatisfiable hints.

    conda config --set unsatisfiable_hints True


    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\exceptions.py", line 1079, in __call__
        return func(*args, **kwargs)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\cli\main.py", line 84, in _main
        exit_code = do_call(args, p)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\cli\conda_argparse.py", line 82, in do_call
        return getattr(module, func_name)(args, parser)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\cli\main_install.py", line 20, in execute
        install(args, parser, 'install')
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\cli\install.py", line 299, in install
        should_retry_solve=(repodata_fn != repodata_fns[-1]),
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\core\solve.py", line 117, in solve_for_transaction
        should_retry_solve)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\core\solve.py", line 158, in solve_for_diff
        force_remove, should_retry_solve)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\core\solve.py", line 281, in solve_final_state
        ssc = self._run_sat(ssc)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\common\io.py", line 88, in decorated
        return f(*args, **kwds)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\core\solve.py", line 808, in _run_sat
        should_retry_solve=ssc.should_retry_solve
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\common\io.py", line 88, in decorated
        return f(*args, **kwds)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\resolve.py", line 1318, in solve
        self.find_conflicts(specs, specs_to_add, history_specs)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\resolve.py", line 347, in find_conflicts
        bad_deps = self.build_conflict_map(specs, specs_to_add, history_specs)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\resolve.py", line 507, in build_conflict_map
        root, search_node, dep_graph, num_occurances)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\resolve.py", line 369, in breadth_first_search_for_dep_graph
        last_spec = MatchSpec.union((path[-1], target_paths[-1][-1]))[0]
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\models\match_spec.py", line 481, in union
        return cls.merge(match_specs, union=True)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\models\match_spec.py", line 475, in merge
        reduce(lambda x, y: x._merge(y, union), group) if len(group) > 1 else group[0]
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\models\match_spec.py", line 475, in <lambda>
        reduce(lambda x, y: x._merge(y, union), group) if len(group) > 1 else group[0]
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\models\match_spec.py", line 502, in _merge
        final = this_component.union(that_component)
      File "C:\Users\Ankur\anaconda3\lib\site-packages\conda\models\match_spec.py", line 764, in union
        return '|'.join(options)
    TypeError: sequence item 0: expected str instance, Channel found

`$ C:\Users\Ankur\anaconda3\Scripts\conda-script.py install -c conda-forge spleeter`

  environment variables:
                 CIO_TEST=<not set>
        CONDA_DEFAULT_ENV=base
                CONDA_EXE=C:\Users\Ankur\anaconda3\condabin\..\Scripts\conda.exe
               CONDA_EXES="C:\Users\Ankur\anaconda3\condabin\..\Scripts\conda.exe"
             CONDA_PREFIX=C:\Users\Ankur\anaconda3
    CONDA_PROMPT_MODIFIER=(base)
         CONDA_PYTHON_EXE=C:\Users\Ankur\anaconda3\python.exe
               CONDA_ROOT=C:\Users\Ankur\anaconda3
              CONDA_SHLVL=1
                 HOMEPATH=\Users\Ankur
                     PATH=C:\Users\Ankur\anaconda3;C:\Users\Ankur\anaconda3\Library\mingw-w64\bi
                          n;C:\Users\Ankur\anaconda3\Library\usr\bin;C:\Users\Ankur\anaconda3\Li
                          brary\bin;C:\Users\Ankur\anaconda3\Scripts;C:\Users\Ankur\anaconda3\bi
                          n;C:\Users\Ankur\anaconda3;C:\Users\Ankur\anaconda3\Library\mingw-w64\
                          bin;C:\Users\Ankur\anaconda3\Library\usr\bin;C:\Users\Ankur\anaconda3\
                          Library\bin;C:\Users\Ankur\anaconda3\Scripts;C:\Users\Ankur\anaconda3\
                          bin;C:\Users\Ankur\anaconda3\condabin;C:\Users\Ankur\anaconda3;C:\User
                          s\Ankur\anaconda3\Library\mingw-w64\bin;C:\Users\Ankur\anaconda3\Libra
                          ry\usr\bin;C:\Users\Ankur\anaconda3\Library\bin;C:\Users\Ankur\anacond
                          a3\Scripts;C:\Program Files (x86)\Intel\Intel(R) Management Engine
                          Components\iCLS;C:\Program Files\Intel\Intel(R) Management Engine
                          Components\iCLS;C:\Program Files (x86)\Common Files\Intel\Shared
                          Files\cpp\bin\Intel64;C:\Program Files (x86)\Common Files\Oracle\Java\
                          javapath;C:\ProgramData\Oracle\Java\javapath;C:\Windows\system32;C:\Wi
                          ndows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v
                          1.0;C:\Windows\System32\OpenSSH;C:\Program Files (x86)\NVIDIA
                          Corporation\PhysX\Common;C:\Program Files (x86)\Intel\Intel(R)
                          Management Engine Components\DAL;C:\Program Files\Intel\Intel(R)
                          Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R)
                          Management Engine Components\IPT;C:\Program Files\Intel\Intel(R)
                          Management Engine
                          Components\IPT;C:\Users\Ankur\AppData\Local\Microsoft\WindowsApps;.
             PSMODULEPATH=C:\Program Files\WindowsPowerShell\Modules;C:\Windows\system32\Windows
                          PowerShell\v1.0\Modules
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : base
    active env location : C:\Users\Ankur\anaconda3
            shell level : 1
       user config file : C:\Users\Ankur\.condarc
 populated config files : C:\Users\Ankur\.condarc
          conda version : 4.8.2
    conda-build version : 3.18.11
         python version : 3.7.6.final.0
       virtual packages : __cuda=10.2
       base environment : C:\Users\Ankur\anaconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\Ankur\anaconda3\pkgs
                          C:\Users\Ankur\.conda\pkgs
                          C:\Users\Ankur\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\Ankur\anaconda3\envs
                          C:\Users\Ankur\.conda\envs
                          C:\Users\Ankur\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.8.2 requests/2.22.0 CPython/3.7.6 Windows/10 Windows/10.0.17763
          administrator : False
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.

If submitted, this report will be used by core maintainers to improve
future releases of conda.
Would you like conda to send this report to the core maintainers?

[y/N]: y
Upload successful.

Thank you for helping to improve conda.
Opt-in to always sending reports (and not see this message again)
by running

    $ conda config --set report_errors true



(base) C:\Users\Ankur>

```

## Environment

|                   |                                 |
| ----------------- | ------------------------------- |
| OS                | Windows  |
| Installation type | Conda  |
| RAM available     | X16GB |
| Hardware spec     | Nvidia GTX 1050 Mini 2GB /Intel i5 7600k @ 3.7GHZ |

**Comment by romi1502** [comment]
Hi @djdecoit,
it seems you have troubles with some already installed dependencies.
Can you try to install spleeter in a clean conda environnement?
```bash 
conda create -n py37 python=3.7
conda activate py37
conda install -c conda-forge spleeter
```

**Comment by tsadler1988** [comment]
I had this problem on a fresh install of Anaconda. I believe it's an issue with Conda https://github.com/conda/conda/issues/9367.

Solved by downgrading Conda to `4.6.14` as recommended on the Conda issue https://github.com/conda/conda/issues/9367#issuecomment-646664751:
```
conda activate base
conda config --set auto_update_conda False
conda install conda=4.6.14
```

**Comment by MyNameIsRene** [comment]
> I had this problem on a fresh install of Anaconda. I believe it's an issue with Conda [conda/conda#9367](https://github.com/conda/conda/issues/9367).
> 
> Solved by downgrading Conda to `4.6.14` as recommended on the Conda issue [conda/conda#9367 (comment)](https://github.com/conda/conda/issues/9367#issuecomment-646664751):
> 
> ```
> conda activate base
> conda config --set auto_update_conda False
> conda install conda=4.6.14
> ```

Quick question: How did you proceede after downgrading conda?
Did you reinstall spleeter with: conda install -c conda-forge spleeter?

**Comment by tsadler1988** [comment]
> Did you reinstall spleeter with: conda install -c conda-forge spleeter?

Yes this command worked for me with conda `4.16.4`.
